### ISSUE TITLE
Component: Dropdown Accessibility

### ISSUE THREAD

**Issue by WTW-Jacob-Evans** [primefaces/primeng#15155]
### Describe the bug

When scanning the dropdown component, we are seeing an error relating to "Scrollable region must have keyboard access"

![image](https://github.com/primefaces/primeng/assets/163871134/2a640a13-89a2-48f8-ac71-8eb35dbf2a0d)
![image](https://github.com/primefaces/primeng/assets/163871134/1f0a04bc-142e-4ad2-bf91-9778255ab7d7)

I understand that this is a feature, and that to remove this warning would interfere with your implementation of keyboard navigation. I attempted to implement a suggestion from your team about adding a template "_Unfortunately, in this situation, we are unable to intervene in the issue. However, to make the test pass, you can use the templating option to add a hidden focusable element to the first element._"; attempting to add this template did not seem to fix the issue, as the issue seems related to the dropdown and not the selected item.

Any further assistance would be appreciated.

### Environment

Windows 10

### Angular version

17.2.3

### PrimeNG version

17.11.0

### Build / Runtime

Angular CLI App

### Language

TypeScript

### Node version (for AoT issues node --version)

18.18.0
